crates/api_common/src/site.rs

```diff
@@ -65,9 +64,10 @@ pub struct GetModlogResponse {
 #[derive(Deserialize)]
 pub struct CreateSite {
   pub name: String,
+  pub sidebar: Option<String>,
   pub description: Option<String>,
-  pub icon: Option<Url>,
-  pub banner: Option<Url>,
+  pub icon: Option<String>,
+  pub banner: Option<String>,
```

Comment:
Why change these back to string?

Reply:
To be able to do these proper overwrites below: https://github.com/LemmyNet/lemmy/pull/1532/files#diff-d9124d8135ec751589f580dd3a84eba50f3ddb2bd27bd2cfb7a436f6af9e9c54R50

Plus the API doesn't actually take in anything but numbers and strings.

Reply:
Ah I see.